### PR TITLE
v2 final

### PR DESCRIPTION
**Pull request by joschka-gross**
Changes made during the re-run of all experiments based on kinodata-3d 2.0.0

### CODE REVIEW

File: kinodata/model/complex_transformer.py [hunk 1]
```diff
@@ -181,6 +182,8 @@ def __init__(
                     max_num_neighbors,
                     hidden_channels,
                 )
+            elif mode == "ligand-structural":
+
```
Comment (mbackenkoehler) [inline kinodata/model/complex_transformer.py:186]: This does probably not belong ;)

File: scripts/generate_dataset.py
Comment (mbackenkoehler) [inline scripts/generate_dataset.py:1]: Maybe add dataset for all rmsd?

File: scripts/generate_splits.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "random-k-fold",
+        "scaffold-k-fold",
+        "pocket-k-fold",
+    )
+    for split_type in split_types:
+        print(f"\tSplitting full dataset based on type {split_type}...")
+        splitter = KinodataKFoldSplit(split_type, 5)
+        splits = splitter.split(dataset)
+
+    for rmsd_cutoff in (2.0, 4.0, 6.0):
```
Comment (mbackenkoehler) [inline scripts/generate_splits.py:38]: Also add 100?

**Review by mbackenkoehler — CHANGES_REQUESTED** [review]
See below

File: scripts/generate_splits.py (hunk 2 above)
Comment (joschka-gross) [reply scripts/generate_splits.py]: Done

File: kinodata/model/complex_transformer.py (hunk 1 above)
Comment (joschka-gross) [reply kinodata/model/complex_transformer.py]: Yes, removed.

**Review by mbackenkoehler — APPROVED** [review]

**Pull request merged**